src/client/replication_ddl_client.cpp

```diff
@@ -1547,5 +1549,36 @@ void replication_ddl_client::query_disk_info(
     call_rpcs_async(query_disk_info_rpcs, resps);
 }
 
+error_with<start_bulk_load_response>
+replication_ddl_client::start_bulk_load(const std::string &app_name,
+                                        const std::string &cluster_name,
+                                        const std::string &file_provider_type)
+{
+    auto req = make_unique<start_bulk_load_request>();
+    req->app_name = app_name;
+    req->cluster_name = cluster_name;
+    req->file_provider_type = file_provider_type;
+    return call_rpc_sync(start_bulk_load_rpc(std::move(req), RPC_CM_START_BULK_LOAD));
+}
+
+error_with<control_bulk_load_response>
+replication_ddl_client::control_bulk_load(const std::string &app_name,
```

Comment:
Which operations does 'control' include, dose it include 'start'?

Reply:
pause, restart, cancel and force cancel, `start` is not included.